protocol/triple/triple-tool/protoc-gen-go-triple/main.go

```diff
@@ -51,19 +53,41 @@ func main() {
 		os.Exit(1)
 	}
 
-	protogen.Options{}.Run(
+	var flags flag.FlagSet
+	useOld := flags.Bool("useOldVersion", false, "print the version and exit")
```

Comment:
@chickenlj We need to talk about this flag to make it more intuitive. And we use ```go-triple_opt=useOldVersion=true``` as temporary placeholder.

Reply:
`go-triple_opt=useOldVersion=true`

This style looks good to me as it's the same as `go-grpc_opt=xxx`